routers/repos.py

```diff
@@ -117,7 +134,7 @@ async def create_repo_docs(
     documentation_service.enqueue_generate_repo_docs_job(
         background_tasks, firestore_repo, model
     )
-
+    background_tasks.add_task(embedding_service.generate_markdown_embeddings_for_repo, firestore_repo.id, user_id)
```

Comment:
instead of adding a second background task, we need to make sure the first one is done before this second one starts running